packages/broadway.rb

```diff
@@ -0,0 +1,79 @@
+require 'package'
+
+class Broadway < Package
+  description 'Run GTK applications in a browser window.'
+  homepage 'http://127.0.0.1:8085'
```

Comment:
This is not a homepage.  This is where you go on your local system to run apps in your browser.  Please change to:
`https://developer.gnome.org/gtk3/stable/gtk-broadway.html`.

Reply:
This should be fair enough. Two things: that page is a moving target, and it's not used in the package.
Why are you so against Chromebrew packages NOT using internet?